ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriberTest.java

```diff
@@ -48,7 +48,7 @@
   private static final ObjectMapper mapper = JsonMapper.INSTANCE.mapper;
 
   static {
-    new SchemaMapper().registerToObjectMapper(mapper);
+    mapper.registerModule(new Jdk8Module());
```

Comment:
ideally we would create a single ObjectMapper, add all the modules in one place, and then inject that into various places we need them, not sure if there is a plan to do that.

Reply:
There actually was a recent change to use a common ObjectMapper - though its accessed as a singleton, not injected. So this line is actually unnecessary. I'll take it out.